docs/src/Makefile

```diff
@@ -32,6 +32,7 @@ help:
 
 clean:
 	-rm -rf $(BUILDDIR)/*
+	-rm -rf generated/*
```

Comment:
In no way!

`generated` is the folder which contains new auto-generated `.rst` files. It ought to be purged every time, otherwise *autosummary* will not update reference page.